doc/releases/changelog-dev.md

```diff
@@ -8,6 +8,9 @@
 
 <h3>Breaking changes 💔</h3>
 
+* The `output_dim` property of `qml.tape.QuantumScript` has been removed. Instead, use method `shape` of `QuantumScript` or `MeasurementProcess` to get the same information.
+  [(#6829)](https://github.com/PennyLaneAI/pennylane/pull/6829)
+
```

Comment:
Don't forget to add yourself as a contributor at the bottom since we're on the next release now :)